### ISSUE TITLE
Unable to install under python3.9

### ISSUE THREAD

**Issue by heinp** [webrecorder/pywb#595]
## Describe the bug

pywb seems to want gevent1.4, but the installation of gevent1.4 doesnt work with python3.9 and raises an error while trying to compile.  So pywb cannot be installed.

## Steps to reproduce the bug

Upgrade to python 3.9
Try to install pywb:
`pip install pywb` or 
```
git clone https://aur.archlinux.org/pywb.git
cd pywb
makepkg -sri
```

## Environment

- OS: Arch Linux
- python version 3.9
- gcc version 10.2
- pywb version 2.4.2

## Error Message
You can take a look at the pip output here: https://pastebin.com/1g8uK3JM

**Comment by heinp** [comment]
A work around seems to be to install all dependencies manually and then install pywb with `pip install --no-deps pywb`.

**Comment by ikreymer** [comment]
Sorry this wasn't released earlier. This should have been fixed by #583 and will be in the upcoming release.

**Comment by ikreymer** [comment]
Now released in 2.5.0!
